pkgs/development/ocaml-modules/raylib/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDunePackage,
+  dune-configurator,
+  ctypes,
+  integers,
+  patch,
+  gitUpdater,
+}:
+
+buildDunePackage rec {
+  pname = "raylib";
+  version = "1.4.0";
+
+  src = fetchFromGitHub {
+    owner = "tjammer";
+    repo = "raylib-ocaml";
+    tag = version;
+    hash = "sha256-Fh79YnmboQF5Kn3VF//JKLaIFKl8QJWVOqRexTzxF0U=";
+    # enable submodules for vendored raylib sources
```

Comment:
Not a blocker, but can we use not-vendored raylib?

Reply:
What should I do for `not-vendored`, I do `patch`?

Reply:
I'm just thinking that if raylib is in nixpkgs (I have not checked) maybe we can use that version. It's not important and maybe we can do it as a follow-up if we want to, it just feels cleaner.